test/commands/lightning/dev/site.test.ts

```diff
@@ -4,39 +4,62 @@
  */
+
+import { strictEqual } from 'node:assert';
 import { TestContext } from '@salesforce/core/testSetup';
-// import { expect } from 'chai';
-// import { stubSfCommandUx } from '@salesforce/sf-plugins-core';
-// import LightningDevSite from '../../../../src/commands/lightning/dev/site.js';
+import { expect } from 'chai';
+import { Org } from '@salesforce/core';
+import { LocalDevOptions } from '@lwrjs/api';
+import LightningDevSite from '../../../../src/commands/lightning/dev/site.js';
+import { OrgUtils } from '../../../../src/shared/orgUtils.js';
 
-// TODO fix me once we have a fully working command
 describe('lightning dev site', () => {
   const $$ = new TestContext();
-  // let sfCommandStubs: ReturnType<typeof stubSfCommandUx>;
 
   beforeEach(() => {
-    // sfCommandStubs = stubSfCommandUx($$.SANDBOX);
+    $$.SANDBOX.stub(OrgUtils, 'isLocalDevEnabled').resolves(true);
+    $$.SANDBOX.stub(OrgUtils, 'ensureMatchingAPIVersion').returns();
   });
 
   afterEach(() => {
     $$.restore();
   });
 
-  it('runs hello', async () => {
-    // await LightningDevSite.run([]);
-    // const output = sfCommandStubs.log
-    //   .getCalls()
-    //   .flatMap((c) => c.args)
-    //   .join('\n');
-    // expect(output).to.include('hello world');
+  it('should have summary, description, and examples defined', () => {
+    strictEqual(typeof LightningDevSite.summary, 'string', 'Summary should be a string');
+    strictEqual(typeof LightningDevSite.description, 'string', 'Description should be a string');
+    strictEqual(typeof LightningDevSite.examples, 'object', 'Examples should be an array');
+  });
+
+  it('result should be undefined if local development is not enabled', async () => {
+    $$.SANDBOX.restore();
+    $$.SANDBOX.stub(OrgUtils, 'isLocalDevEnabled').resolves(false);
+    const result = await LightningDevSite.run(['--name', 'Astro', '--target-org', '00Dxx0000001gEH']);
+    expect(result).to.be.undefined;
   });
 
-  it('runs hello world --name Astro', async () => {
-    // await LightningDevSite.run(['--name', 'Astro']);
-    // const output = sfCommandStubs.log
-    //   .getCalls()
-    //   .flatMap((c) => c.args)
-    //   .join('\n');
-    // expect(output).to.include('hello Astro');
+  it('should have valid startupParams', async () => {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-argument, @typescript-eslint/no-explicit-any
+    const org = new Org({ id: '00Dxx0000001gEH', connection: {} } as any);
+    $$.SANDBOX.stub(Org, 'create').returns(Promise.resolve(org));
+
+    const startupParams: LocalDevOptions = {
+      sfCLI: true,
+      authToken: 'test-auth-token',
+      open: true,
+      port: 3000,
+      logLevel: 'error',
+      mode: 'dev',
+      siteZip: 'test-site-zip',
+      siteDir: 'test-site-dir',
+    };
+
+    $$.SANDBOX.stub(LightningDevSite, 'run').resolves(Promise.resolve(startupParams));
```

Comment:
This test isn't testing anything. You are mocking the entire run method and having it always return startupParams. So you are just testing if(startupParams === startupParams).